pkg/plugin/group/scaler.go

```diff
@@ -118,6 +118,7 @@ func (s *scaler) PlanUpdate(scaled Scaled, settings groupSettings, newSettings g
 		scaled:       scaled,
 		updatingFrom: settings,
 		updatingTo:   newSettings,
+		desiredSize:  int(settings.config.Allocation.Size),
```

Comment:
if we initialize this value here...  does this ever get updated when we change the count (Allocation.Size) of a group?